src/features/user/DonationReceipt/microComponents/ReceiptDataSection.tsx

```diff
@@ -70,20 +70,28 @@ const ReceiptDataSection = ({ donationReceiptData }: Prop) => {
     year,
     updateDonationReceiptData,
   ]);
+  const { address1, zipCode, city, country } = address;
   const hasMultipleDonations = donations.length > 1;
+  const isAddressInvalid = !address1 || !zipCode || !city || !country;
+  const isDonorContactInvalid = isAddressInvalid || !donor.name;
   return (
```

Comment:
Name `isDonorContactInvalid` is a bit confusing - just call it `isContactInfoInvalid`.